conans/client/generators/scons.py

```diff
@@ -0,0 +1,37 @@
+from conans.model import Generator
+
+class SConsGenerator(Generator):
+    @property
+    def filename(self):
+        return "SConscript-conan"
```

Comment:
Is the intent that the file would be picked up by scons automatically? if so then the following from the manpage applies "

> By default, scons searches for a file named SConstruct, Sconstruct, or sconstruct (in that order) in the current directory and reads its configuration from the first file found. An alternate file name may be specified via the -f option.